### ISSUE TITLE
Other players can pack up siege engines and run off with them, whilst a player or his npc's are using them

### ISSUE THREAD

**Issue by SyndicMohawky** [P3pp3rF1y/AncientWarfare2#1280]
When using a siege engine in multiplayer, its possible for a non-allied player to sneak up and pack up the siege engines (ballistae, rams etc) that a player is currently sat on and operating, and just run off with it.

This is an exploitative cheat.

Can we somehow make siege engines unpackable if a non-allied or enemy player or npc is currently using it?

**Comment by retard153453465** [comment]
actual brain damaged developer, how could you release something with this flaw and think its ok

**Comment by kirbcake** [comment]
Yeah, this is quite dumb. especially if it was a really expensive siege engine, or is a battering ram that is mostly unarmed. My best suggestion for now is to have a large soldier escort or another player protecting you.
